src/systems/filecoin_vm/actor/address/address.go

```diff
@@ -55,6 +55,11 @@ func (a *Address_I) IsKeyType() bool {
 	panic("TODO")
 }
 
+func (a *Address_I) ToBytes() util.Bytes {
```

Comment:
`addr.Serialize_Address_Compact` should be used here, no need for a new method I think.